pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -258,15 +255,18 @@ func (ca *cloudCIDRAllocator) AllocateOrOccupyCIDR(node *v1.Node) error {
 }
 
 // updateCIDRAllocation assigns CIDR to Node and sends an update to the API server.
-func (ca *cloudCIDRAllocator) updateCIDRAllocation(nodeName string) error {
-	node, err := ca.nodeLister.Get(nodeName)
+// Operate on the `node` object if any changes have to be done to it in the API.
```

Comment:
The golang style guide suggests that the comment should start with the function name "updateCIDRAllocation operates on the...": See the 'Funcs' section here: https://tip.golang.org/doc/comment

Reply:
It does starts with the func name, see line above.